### PR TITLE
add news and newsDetails page

### PR DESCRIPTION
**Pull request by Light111989**

### CODE REVIEW

File: src/components/news.vue [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<template>
+  <q-carousel class="component-top" style="max-width: 700px;margin:auto; height:300px">
+    <q-carousel-slide :img-src="item.img">
+      <div class="absolute-top-left custom-caption-top">
+        <div class="subtitle text-white q-ml-sm q-mt-sm" >{{item.sub}}</div>
+      <div class="q-display-1 text-white q-ml-sm q-mb-sm" >{{item.title}}</div>
+        <q-btn class=" fixed-bottom-left q-ml-sm q-mb-sm" :class="{ hidden: item.isBtnHidden }" color="white" text-color="brown"  rounded>
+          <q-item  @click.native="show">
```
Comment (hemvanh) [inline src/components/news.vue]: what is the purpose of .native ?

File: src/components/news.vue [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        </q-btn>
+        <slot></slot>
+      </div>
+    </q-carousel-slide>
+  </q-carousel>
+</template>
+<script>
+export default {
+  props: {
+    item: [Object],
```
Comment (hemvanh) [inline src/components/news.vue]: [Object]  -> this means item is an array of Objects ?

File: src/components/newsDetails.vue [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+          <div>Order</div>
+        </q-btn>
+      </div>
+    </q-modal-layout>
+  </q-modal>
+</template>
+<script>
+export default {
+  props: {
+    item: [Object],
```
Comment (hemvanh) [inline src/components/newsDetails.vue]: [Object] -> this means item is an array of Objects ?

File: src/pages/home.vue [hunk 4]
```diff
@@ -1,177 +1,55 @@
...
-      <q-carousel-slide img-src="~assets/panel.jpg">
-      </q-carousel-slide>
-    </q-carousel>
-    <q-carousel>
-      <q-modal v-model="opened">
-        <h4>Basic Modal</h4>
-        <q-btn color="primary" @click="opened = false" label="Close" />
-      </q-modal>
-    </q-carousel>
+    <news v-for="(item,index) in news" :key="`news${index}`" :item="item">
```
Comment (hemvanh) [inline src/pages/home.vue]: :key="`news${index}`"  -->change to -->  :key="index" ?

File: src/pages/home.vue [hunk 5]
```diff
@@ -1,177 +1,55 @@
...
-      imageSrc: '~assets/component.jpg',
-      types: [
+      news: [
         {
+          img: 'assets/component.jpg',
           label: 'Learn More',
-          show: () => {
-            this.basicModal = true
-          },
+          isBtnHidden: false,
```
Comment (hemvanh) [inline src/pages/home.vue]: swap properties' name and order to: (for all objects)
          img:
          isBtnHidden: 
          label:
          title:
          subTitle: 
          titleContent: 
          content:

File: src/components/news.vue (hunk 1 above)
Comment (Light111989) [reply src/components/news.vue]: when we use <q-item> must add  (native)
but this is redundant. i removed it

**Pull request merged**